packaging/rpm/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM fedora:latest
+
+RUN dnf install -y git make bash rpm-build rpmdevtools 'dnf-command(builddep)'
```

```suggestion
RUN dnf install -y git make bash rpm-build 'dnf-command(builddep)'
```
I haven't tested it, but I think you can get rid of `rpmdevtools` since you removed `rpmdev-setuptree` from the Makefile. If needed it will be pulled in later by `dnf builddep`.